webapp/src/lib/WebSocketClient.js

```diff
@@ -188,6 +190,11 @@ class WebSocketClient extends EventEmitter {
 		}
 	}
 
+	_handleReload (message) {
+		this._reconnect = false;
```

Comment:
I'd rather just normally close the connection instead of an extra flag.

Reply:
makes sense, I wasn't sure if that would trigger the reconnect again

Reply:
… or cause a reconnect if the server is closing the connection quicker than we are ;) but I can stop the server from closing after this message as ewll

Reply:
If the server closes the connection before the message is handled it's too late anyway. Not sure if the browser does those in order.